packages/vuetify/src/components/VAppBar/VAppBar.tsx

```diff
@@ -0,0 +1,113 @@
+// Styles
+import './VAppBar.sass'
+
+// Composables
+import { makeBorderProps, useBorder } from '@/composables/border'
+import { makeDensityProps, useDensity } from '@/composables/density'
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makeLayoutItemProps, useLayoutItem } from '@/composables/layout'
+import { makePositionProps, usePosition } from '@/composables/position'
+import { makeRoundedProps, useRounded } from '@/composables/rounded'
+import { makeTagProps } from '@/composables/tag'
+// import { useDisplay } from '@/composables/display'
+// import { useProxiedModel } from '@/composables/proxiedModel'
+import { useTheme } from '@/composables/theme'
+import { useBackgroundColor } from '@/composables/color'
+
+// Utilities
+import { makeProps } from '@/util/makeProps'
+import { computed, defineComponent, toRef } from 'vue'
+
+export default defineComponent({
+  name: 'VAppBar',
+
+  props: makeProps({
+    color: String,
+    flat: Boolean,
+    height: [Number, String],
+    modelValue: Boolean,
+    prominent: Boolean,
+    src: String,
+    temporary: Boolean,
+    ...makeBorderProps(),
+    ...makeDensityProps(),
+    ...makeElevationProps(),
+    ...makeLayoutItemProps(),
+    ...makePositionProps(),
+    ...makeRoundedProps(),
+    ...makeLayoutItemProps({ name: 'app-bar' }),
```

Comment:
You use this twice.

Reply:
also, `makeLayoutItemProps` is giving you `size` prop but you also have `height`